File: Bimestre1/Tarea4/Tarea4Python/alumnos.py
print("Para leer las calificaciones de N alumnos y determine el número de aprobados y reprobados.")
# inicialización de variables
alum = int(1)
aprob = int(0)
reprob = int(0)
nota = float(0)
# Entrada de datos.
n = int(input("Ingrese la cantidad de alumnos a evaluar: "))
nota = int(input("Ingrese la nota del alumno: "))
if nota < 7:
        reprob = reprob + 1
else:
        aprob = aprob + 1
while alum < n:
    nota = int(input("Ingrese la nota del alumno: "))
    if nota < 7:
        reprob = reprob + 1
    else:
        aprob = aprob + 1
    alum = alum + 1
print("La cantidad de alumnos que probaron fueron: ",aprob)
print("La cantidad de alumnos que reprobaron fueron: ",reprob)

File: Bimestre1/Tarea4/Tarea4/src/tarea4/alumnos.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package tarea4;
import java.util.Scanner;
/**
 *
 * @author kevin
 */
public class alumnos {
    public static void main(String[] args) {
        System.out.println("para leer las calificaciones de N alumnos y "
                + "determine el número de " +
                "aprobados y reprobados. ");
        Scanner leer = new Scanner(System.in);
        //creación e inicialización de variables
        int n, alum,aprob,reprob;
        double nota;
        n=0;alum=1;aprob=0;reprob=0; nota=0;
        //entrada de datos
        System.out.print("Ingrese la cantidad de alumnos a evaluar: ");
        n = leer.nextInt();
        while(alum<=n){
            System.out.println("Ingrese la nota del alumno:");
            nota = leer.nextDouble();
            if(nota<7){
                reprob=reprob+1;
                  
            }
            else{
                aprob=aprob+1;
            }
            alum=alum+1;
        
        }
        System.out.println("La cantidad de alumnos que probaron fueron: "+aprob);
        System.out.println("La cantidad de alumnos que reprobaron fueron: "+reprob);
        
    }
    
}
